OKTA-438759 Convert configure-access-policy to single page guide (#2871)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md

---
title: Configure an access policy
meta:
  - name: description
    content: Okta access policies help you secure your APIs by defining different access and refresh token lifetimes for a given combination of grant type, user, and scope.
layout: Guides
---

This guide helps you understand access policies and how to configure them for common scenarios.

---
<!-- Nutrition facts bullets -->
**Learning outcomes**

* Understand access policies.

> * Configure an access policy to limit which scopes some clients can access.

Reviewer: ```suggestion
* Configure an access policy to limit the scopes that some clients can access.
```

> * Configure an access policy to limit which scopes some clients can access.

Reviewer: ```suggestion
* Configure an access policy to limit which scopes that some clients can access.
```

This guide provides step-by-step instructions to configure an access policy for two of the most common scenarios:

* [Limit which scopes some clients can access](#limit-which-scopes-some-clients-can-access)
* [Configure a custom token lifetime per client](#configure-a-custom-access-token-lifetime-per-client)

## Limit which scopes some clients can access

Use the following steps to create a policy that limits which scopes that a designated client can access.

### Create the policy container

1. In the Admin Console, go to **Security** > **API**.

2. On the **Authorization Servers** tab, select the name of the authorization server that you want to apply the policy to.

3. Select **Access Policies**, and then **Add Policy**.

4. Enter a **Name** and a **Description** for the policy.

5. Select **The following clients:** and start typing the names of the Okta OpenID Connect applications that you want to cover with the access policy. This field automatically displays a list of applications that match what you type.

6. Click **Create Policy**.

### Create the rule

1. Select the name of the access policy that you just created, and then select **Add Rule**.

> 2. Enter a **Name** for the rule, and then for this example leave the defaults for **IF Grant type is** and **AND User is**.

Reviewer: ```suggestion
2. Enter a **Name** for the rule, and then for this example, leave the defaults for **IF Grant type is** and **AND User is**.
```

---
title: Configure an access policy
meta:
  - name: description
    content: Okta access policies help you secure your APIs by defining different access and refresh token lifetimes for a given combination of grant type, user, and scope.
layout: Guides
---

This guide helps you understand access policies and how to configure them for common scenarios.

---
<!-- Nutrition facts bullets -->
**Learning outcomes**

* Understand access policies.
* Configure an access policy to limit which scopes some clients can access.
* Configure a custom access token lifetime per client.

**What you need**

* [Okta Developer Edition organization](https://developer.okta.com/signup)
* An OIDC client app for testing access policies

<ApiAmProdWarning />

---

> ## Overview

Reviewer 2: It's not in the Confluence doc, but I think that Christine prefers that we avoid using "Overview" here. Something like "About configuring access policies" could be more informative.
